conf/app.routes

```diff
@@ -173,4 +173,7 @@ GET        /change-agent-telephone-number                   controllers.AgentTel
 POST       /change-agent-telephone-number                  controllers.AgentTelephoneNumberController.onSubmit(mode: Mode = CheckMode)
 
 GET        /agent-overview                                      controllers.AgentOverviewController.onPageLoad()
+
 GET        /trustees-information                                controllers.TrusteesInfoController.onPageLoad()
+
+GET        /asset-interrupt-page                       controllers.AssetInterruptPageController.onPageLoad()
```

Comment:
@JoshuaH01  Is it ok to make asset/interrupt-page? May be discuss with @adamconder

Reply:
I'd probably suggest something along the lines of /assets/information. Can we speak with Annita on this?

Reply:
Or event `assets/information-you-need` as per the title.